scripts/end_to_end_test/run_end_to_end_test.py

```diff
@@ -620,20 +620,23 @@ def run_test(build_directory, yaml_file, node_exe, name_filter=None):
             for test in all_yaml:
                 # Get test setup conditions
                 setup_conditions = yaml_extract(test, 'setup_conditions')
+                description = yaml_extract(test, 'test_description')
+
+                test_disabled = "DISABLED" in description
 
-                # Check if name is not filtered out
+                # Check if name is not filtered out (overrides disabled specifier)
```

Comment:
I don't think this is a good idea. If someone is working on a test, they might as well re-enable it in the YAML. This creates a risk where someone will locally fix a disabled test and commit changes without removing the `DISABLED` tag. This mistake would not be obvious from the GitHub diff and thus would be easy to miss in review.

Reply:
I can see this happening and I guess it is not much effort to enable the test